tests/fixtures/clients.py

```diff
@@ -0,0 +1,14 @@
+from typing import TYPE_CHECKING
+
+import pytest
+from starlette.testclient import TestClient
+
+if TYPE_CHECKING:
+    from fastapi import FastAPI
+
+
+@pytest.fixture
+def unauthenticated_client(
```

Comment:
```suggestion
def client(
```
we don't have more clients for now, rule of thumb